src/core_plugins/kibana/public/dashboard/dashboard_config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Part of the exposed plugin API - do not remove without careful consideration.
+ * @type {boolean}
+ */
+let hideDashboardWriteControls = false;
```

Comment:
We can't store state in module scope or multiple instances of the app will be sharing a configuration when they shouldn't be